packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHeadersHidden.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useCallback } from 'react';
+import { FlowsheetDataGrid } from 'terra-data-grid';
+
+const ColumnHeadersHidden = () => {
+  const gridDataJSON = {
+    cols: [
+      { id: 'Column-0', displayName: 'Vitals', isSelectable: false },
```

Comment:
Column headers shouldn't be selectable when hidden.
That could be done here by setting isSelectable to false on all the headers, but it would be better if it were a hard constraint of the component.

Reply:
Agreed, in testing the example when in the first cell has focus I can up arrow once, right arrow, and then down arrow and the focus is in the cell to the right (however many time I hit the right arrow) I wouldn't expect you'd be able to navigate to and through them.

Reply:
Fixed here: 3dfb79bf7ca0dbd76593a7dcd3cea5889a98b113